notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -170,7 +170,65 @@ asteroid impact [Sherman Klump had a dream in which a giant asteroid collided wi
 father and son [Sherman Klump had a heart to heart talk with his father Cletus about getting back together with Denise.]
 
 :: Other Keywords
-giant hamster
+giant hamster []
+
+
+movie: The 6th Day (2000)
+=========================
+
+:: Title
+The 6th Day
+
+:: Description
+The 6th Day is a 2000 American science fiction action film directed by Roger
+Spottiswoode and starring Arnold Schwarzenegger, Tony Goldwyn, Michael
+Rapaport and Robert Duvall.
+
+Synopsis: In the film, a family man of the future is illegally cloned by
+accident as part of a vast conspiracy involving a shady billionaire
+businessman, and is thrust into a struggle to clear his name and protect his
+family from the conspirators who seek to keep the cloning a secret. The title
+refers to the Christian Genesis creation narrative, where God created mankind
+on the sixth day.
+
+:: Date
+2000-10-28
+
+:: References
+https://en.wikipedia.org/wiki/The_6th_Day
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+human cloning [The story is set in a near future when human cloning is possible, but strictly forbidden by law. The story itself follows the family man Adam Gibson getting cloned by a nefarious corporation and its action packed aftermath.]
+human cloning in society [The film confronts the viewer with the question of whether human cloning should be legal. In the film, anti-cloning activists thought it dangerous and possibly an abomination onto the Lord, while the cloning corporation Replacement Technologies tried to make the case for cloning. In the end, the Adam clone came to take a middle position on the issue.]
+
+:: Major Themes
+a person and their clone [The Adam clone teamed up with the real article to free Adam's wife and daughter from the clutches of a nefarious corporation.]
+husband and wife [Central to the story were Adam and Natalie Gibson. Dr. Griffin Weir and his terminally ill clone wife Katherine.]
+unethical business practices [Replacement Technologies was secretly cloning people left, right and center in flagrant violation of the human cloning prohibiting "Sixth Day" laws.]
+what if I met my double [The Adam clone teamed up with the real article in order to rescue Adam's wife and daughter from the clutches of a ruthless billionaire CEO.]
+the quest for immortality [The CEO of Replacement Technologies Michael Drucker was engaged in nefarious research into human cloning because he wanted to conquer death; in particular his own.]
+familial love [The viewer is shown just how much both the Adam clone and the real article loved Adam's wife and young daughter.]
+human self improvement [The story culminated with both Adam and the Adam clone changing their view on cloning in general, and human cloning in particular. They went from thinking clones were weird and creepy, to accepting that clones were people too.]
```

Comment:
This doesn't sound right. The comment suggests that the Themer thinks the characters improved. But the theme in question is supposed to be about the characters thinking about themselves and trying to improve.

Reply:
Fair enough. I've deleted it - although future themers may dig up proper justification for it.

Reply:
Note: I added **playing God with nature**.